docs/tables/twitter_search_recent.md

```diff
@@ -1,4 +1,4 @@
-# Table: twitter_search
+# Table: twitter_search_recent
```

Comment:
@judell There's also an old reference in README.md I believe, can you please update that one as well?

Reply:
Done